Added more logos

l-marchesi opened the pull request:
Hi @jamie-mh 

I've added 12 more logos.

jamie-mh commented:
Hi @l-marchesi ,

Thanks for working on these icons. Everything looks good.
However, it seems like BitBay recently rebranded as [Zonda](https://zondaglobal.com). Can you please update that icon.

Thanks

l-marchesi commented:
Hi @jamie-mh 

You're welcome :)
I've deleted the BitBay logo and replaced it with the Zonda logo. Since I don't speak 'Czech' I added the "dark variant by inverting the colors.

jamie-mh commented:
Thanks. Looks good, I'll merge.
